website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -72,7 +73,7 @@ Please read through the entire procedure before beginning the upgrade. There are
 
    ```bash
    helm upgrade --install karpenter-crd oci://public.ecr.aws/karpenter/karpenter-crd --version "${KARPENTER_VERSION}" --namespace "${KARPENTER_NAMESPACE}" --create-namespace \
-        --set webhook.enabled=true \
+        --set webhook.enabled=false \
```

Comment:
I thought we were saying that we could just go straight into upgrading to the v1 with webhook enabled?
```suggestion
        --set webhook.enabled=false \
```

Reply:
Yeah, you can. This is proposing that you can upgrade the CRDs and without creating downtime to the pre-v1 controller.

Reply:
I'm missing the point of why'd we want to propose this? Seems like we'd force them to do more upgrades. We should just recommend the safest shortest path

Reply:
We tell them to disable the webhooks in the CRD, not in the controller. When they run the helm upgrade for the controller with the webhooks enabled the post install hook will add the conversion block. So no extra steps needed from users

Reply:
This feels like one-too-many steps here. We shouldn't need to go from disabling webhooks to enabling webhooks like this IMO, seems like an extra step that really isn't necessary to avoid downtime